### DISCUSSION TITLE
Can the spam filtering function  filterInput(field, event) and spam keyword definitions be/need to be rewritten as components?

### DISCUSSION THREAD

**Discussion by leechiuhui** [3dw/auto20-next#136] [Q&A] [answered]
First, record and inquire. Plan to research, consider, and address it on 7/5 (next Friday).

**Comment by bestian** [comment, marked as answer]
不需要放成component, 寫成一個獨立的keywords.js檔然後import它就可以方便編輯了

**Comment by leechiuhui** [comment]
OKkkk......now work on other files to add  import { keywords } from '../data/keywords.js';
